cmd/limactl/copy.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"os/user"
+	"strings"
+
+	"github.com/AkihiroSuda/lima/pkg/sshutil"
+	"github.com/AkihiroSuda/lima/pkg/store"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+)
+
+var copyCommand = &cli.Command{
+	Name:        "copy",
+	Aliases:     []string{"cp"},
+	Usage:       "Copy files between host and guest",
+	Description: "Prefix guest filenames with the instance name and a colon.\nExample: limactl copy default:/etc/os-release .",
+	ArgsUsage:   "SOURCE ... TARGET",
+	Action:      copyAction,
+}
+
+func copyAction(clicontext *cli.Context) error {
+	if clicontext.NArg() < 2 {
+		return errors.Errorf("requires at least 2 arguments: SOURCE DEST")
+	}
+	arg0, err := exec.LookPath("scp")
+	if err != nil {
+		return err
+	}
+	u, err := user.Current()
+	if err != nil {
+		return err
+	}
+
+	args, err := sshutil.SCPArgs()
+	if err != nil {
+		return err
+	}
+	args = append(args, "-3", "--")
+	for _, arg := range clicontext.Args().Slice() {
```

Comment:
We may want to limit the length of the slice to be 2?

Reply:
Why? You can specify multiple source arguments when the destination is a directory:

```console
$ limactl cp alpine:/etc/os-release default:/etc/hostname opensuse:.
jan@mactop:~/suse/lima [readme %]
$ limactl shell opensuse sh -c 'ls -l ~'
total 12
drwxr-xr-x 2 jan users   6 Mar  3 11:52 bin
-rw-r--r-- 1 jan users  13 Jun 29 07:47 hostname
-rw-r--r-- 1 jan users 164 Jun 29 07:47 os-release
```

Reply:
Ah, that's awesome :+1: